examples/01_basic/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"reflect"
+
+	"github.com/taxibeat/harvester/config"
+	"github.com/taxibeat/harvester/seed"
+)
+
+// ConfigAttrs sample config struct
+type ConfigAttrs struct {
+	Name string `seed:"John Doe"`
+	Age  int64  `seed:"18" env:"ENV_AGE"`
+}
+
+func main() {
+	attrs := ConfigAttrs{}
+
+	cfg, err := config.New(&attrs)
+	if err != nil {
+		fmt.Printf("Oops, something went wrong creating config: %v", err)
+	}
+
+	printAttrs(attrs)
+
+	s := seed.New()
```

Comment:
You should use harvester and not the individual components for config and seed.

Reply:
👍